android/app/src/main/assets/capacitor.config.json

```diff
@@ -7,7 +7,9 @@
   "plugins": {
     "IntercomPlugin": {
       "android-apiKey": "android_sdk-xxxx",
-      "android-appId": "yyyy"
+      "android-appId": "yyyy",
+      "ios-apiKey": "ios_sdk-xxxx",
```

Comment:
probably don't need to add the ios stuff (or anything) here if it gets overridden by the top level capacitor.config

Reply:
That's the reason why that file has this content (the top-level config file has it).